### PR TITLE
Removing deepsea notes from troubleshooting guide

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

File: xml/bp_troubleshooting_tips.xml [hunk 1]
```diff
@@ -318,23 +318,21 @@ osd_scrub_load_threshold = 0.25
 
   <tip>
    <para>
-    Create the <filename>/var/lib/ceph</filename> subvolume before running
-    &deepsea; stage 0 because stage 0 installs &ceph; related packages and
-    creates the <filename>/var/lib/ceph</filename> directory.
+    Create the <filename>/var/lib/ceph</filename> subvolume before installing
```
Comment (tbazant) [inline xml/bp_troubleshooting_tips.xml:321]: This whole section is IMO a good candidate for removal as MONs (and all services) run in containers in SES7

**Comment by asettle** [comment]
> This looks unfinished. I hope that is intentional!

This is not an update for an edit. It's just a removal of outdated content. I'll be closing your review at this time :)

File: xml/bp_troubleshooting_tips.xml (hunk 1 above)
Comment (asettle) [reply xml/bp_troubleshooting_tips.xml:321]: Okay, I'll remove this section and add a note for future Alex.

File: xml/bp_troubleshooting_tips.xml [hunk 2]
```diff
@@ -1294,44 +954,8 @@ ID WEIGHT TYPE NAME    UP/DOWN REWEIGHT PRIMARY-AFFINITY
...
-    10360.0 Mbits/sec
-192.168.128.3:
-    9336.0 Mbits/sec
-192.168.128.4:
-    9588.56 Mbits/sec
-192.168.128.5:
-    10187.0 Mbits/sec
-192.168.128.6:
-    10465.0 Mbits/sec</screen>
+     NOTE: - Cephsalt needs to implemtent network diagnostics.
```
Comment (tbazant) [inline xml/bp_troubleshooting_tips.xml:957]:
```suggestion
     NOTE: - &cephsalt; needs to implement network diagnostics.
```

**Pull request merged**